package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hyperview",
-  "version": "0.40.0",
+  "version": "0.40.1",
```

Comment:
FWIW, we don't need to update the NPM module, the schema is not bundled as part of the module but rather published via the Hyperview website as soon as the master branch is updated.

Reply:
Oh shoot, just saw this after I merged. Alright, good to know for any future changes.

Reply:
To note also: version number should only be bumped with our versioning script, so that this ref and NPM don't get out of sync. In this present case, it's not too big of a deal, we'll just effectively skip 0.40.1 version when we patch release next.

Reply:
Ohhh fancy! I like it. I'm used to just doing it myself.